src/Maps/Map.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import GoogleMapReact from 'google-map-react';
+import { settings } from '../settings';
+import { MapMarker, MapMarkerProps } from './MapMarker';
+
+interface MapProps {
+  center?: {
+    lat?: number
+    lng: number
+  },
+  zoom?: number
+}
+
+export const Map = (props: MapProps) => {
+
+  const [cursorPosition, setCursorPosition] = useState<MapMarkerProps>({
+    lat: Number(props.center?.lat),
+    lng: Number(props.center?.lng)
+
+  });
+
+
+  return (
+    <div style={{ height: '80%', width: '100%' }}>
```

Comment:
Same comment on using inline styles. If you're gonna use CSS classes be consistent